cmake/modules/grpc.cmake

```diff
@@ -25,35 +25,46 @@ elseif(NOT USE_BUNDLED_GRPC)
 else()
 	include(cares)
 	include(protobuf)
+	include(openssl)
 	find_package(PkgConfig)
 	if(NOT PKG_CONFIG_FOUND)
 		message(FATAL_ERROR "pkg-config binary not found")
 	endif()
 	set(GRPC_SRC "${PROJECT_BINARY_DIR}/grpc-prefix/src/grpc")
 	set(GRPC_INCLUDE "${GRPC_SRC}/include")
-	set(GRPC_LIB "${GRPC_SRC}/libs/opt/libgrpc_unsecure.a")
-	set(GRPCPP_LIB "${GRPC_SRC}/libs/opt/libgrpc++_unsecure.a")
+	set(GRPC_LIB "${GRPC_SRC}/libs/opt/libgrpc.a")
+	set(GRPCPP_LIB "${GRPC_SRC}/libs/opt/libgrpc++.a")
```

Comment:
Why do we switch libraries here? I'm not necessarily opposed to it ("unsecure" is bad, right?) but it needs more comment than "version bump"

Reply:
AFAIK, the "unsecure" flavor exists to avoid versioning hell with OpenSSL. Since we don't have this issue, the natural choice was to switch to the "normal" flavor. Although that should have no impact on the libs, it might help libs implementers. Btw, the "normal" grpc lib flavor is used by Falco, and it has been tested to work in real scenarios.

Anyway, I have forgotten to switch the gRCP libraries for the case when gRPC is not embedded:
https://github.com/falcosecurity/libs/blob/09f8f60667acfed579001458e455b8db62b32e2b/cmake/modules/grpc.cmake#L14-L15

I will fix it